radio/src/gui/480x272/colors.h

```diff
@@ -28,8 +28,11 @@
 #define TO4BITS(x)                     (uint32_t(limit<int>(0, x, 255)) >> 4)
 #define TO5BITS(x)                     (uint32_t(limit<int>(0, x, 255)) >> 3)
 #define TO6BITS(x)                     (uint32_t(limit<int>(0, x, 255)) >> 2)
-#define RGB(r, g, b)                   ((TO5BITS(r) << 11) + (TO6BITS(g) << 5) + (TO5BITS(b) << 0))
+#define RGB_SLOW(r, g, b)                   ((TO5BITS(r) << 11) + (TO6BITS(g) << 5) + (TO5BITS(b) << 0))
 #define ARGB(a, r, g, b)               ((TO4BITS(a) << 12) + (TO4BITS(r) << 8) + (TO4BITS(g) << 4) + (TO4BITS(b) << 0))
+
+#define RGB(r, g, b)                   (uint16_t)((((r) & 0xF8) << 8) + (((g) & 0xFC) << 3) + (((b) & 0xF8) >> 3))
+
```

Comment:
perhaps we could use a new LIMIT definition here, I think it should work for both RGB and ARGB